bom/application/pom.xml

```diff
@@ -46,12 +46,12 @@
         <microprofile-rest-client.version>4.0</microprofile-rest-client.version>
         <microprofile-jwt.version>2.1</microprofile-jwt.version>
         <microprofile-lra.version>2.0</microprofile-lra.version>
-        <microprofile-openapi.version>3.1.2</microprofile-openapi.version>
+        <microprofile-openapi.version>4.0.2</microprofile-openapi.version>
         <smallrye-common.version>2.8.0</smallrye-common.version>
         <smallrye-config.version>3.10.0</smallrye-config.version>
         <smallrye-health.version>4.1.0</smallrye-health.version>
         <smallrye-metrics.version>4.0.0</smallrye-metrics.version>
-        <smallrye-open-api.version>3.13.0</smallrye-open-api.version>
+        <smallrye-open-api.version>4.0.0-beta2</smallrye-open-api.version>
```

Comment:
This is just for testing purposes right? (and the reason the PR is a draft I guess)

Reply:
Yes, exactly. The 4.0.0 release should be out in a a few days but it won't really change much about this PR other than this version property.